src/pages/WordFrequency/components/DiffView/DiffView.tsx

```diff
@@ -65,42 +67,45 @@ export const DiffView = (props: DiffViewProps) => {
     fetchData()
   }, [nodeRelativePath, script])
 
-  const columnDefinitions: KSCUI.C.KsTableColumnDefinition[] = [

```

Comment:
general suggestion is to try a reduce the diffs in changes. 

it is not just so the PR is easier but also make is easy (and therefore more likely) that the author can ensure only changes he intended go in.

i thin